build

```diff
@@ -18,10 +18,6 @@ fi
 
 echo Building kube-aws ${VERSION}
 
-go generate ./core/controlplane/config
-go generate ./core/nodepool/config
-go generate ./core/root/config
```

Comment:
Hi! Thank you very much for your interest in kube-aws and the efforts.

Basically, I appreciate it if we could keep our existing workflow of just editing separate template files rather than battling against multiline string vars.

But on the other hand, I do believe that your use-case is reasonable. So, I'd like to discuss with you to find a sweetspot that works for both of us!

That being said, my suggestion is: how about replacing these `go generates` with something like:

```bash
# Instead of `go generate ./core/controlplane/config
pushd core/controlplane/config/templates
go run ../../../codegen/templates_gen.go CloudConfigController=cloud-config-controller CloudConfigWorker=cloud-config-worker CloudConfigEtcd=cloud-config-etcd DefaultClusterConfig=cluster.yaml KubeConfigTemplate=kubeconfig.tmpl StackTemplateTemplate=stack-template.json
gofmt -w templates.go
go run ../../../codegen/files_gen.go Etcdadm=../../../etcdadm/etcdadm
gofmt -w files.go
popd

# And repeat the above pattern for `./core/nodepool/config` and `./core/root/config` respectively
```

and DO commit generated files to git?

Reply:
yeah, there is a disadvantage in having templates as content of variable in go (formatting, escaping, no native editor support - there's no way for editor to know if the string content of the variable is yaml or json or bash ...)

so yeah, I'm happy with your solution 👍

Reply:
I'm happy to do it (remove from .gitignore, update make, create pull request) on Monday